qa-ibd.planx-pla.net/portal/gitops.json

```diff
@@ -95,17 +102,23 @@
           "name": "Query"
         },
         {
-          "name": "Notebook Browser",
+          "icon": "query",
           "link": "/resource-browser",
           "color": "#a2a2a2",
-          "icon": "query"
+          "name": "Notebook Browser"
         },
         {
           "icon": "workspace",
           "link": "#hostname#workspace/",
           "color": "#a2a2a2",
           "name": "Workspace"
         },
+        {
+          "icon": "query",
```

Comment:
did you have a look at the existing icons to see if we can use something different, since we're using the "query" icon 3 times?